src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -133,10 +130,8 @@ def _validate_individual_class_sheets(class_df_dict: dict[str, pd.DataFrame]) ->
         return [ExcelSheetProblem(sheet, [missing]) for sheet, missing in missing_required_columns.items()]
     problem_list: list[PositionInExcel] = []
```

Comment:
Here again, I think that the naming is not optimal: These are actually not problems, but rather positions, so I would vote for naming them `positions_of_problems` or so